Library/Homebrew/extend/ARGV.rb

```diff
@@ -120,6 +120,10 @@ def dry_run?
     include?("--dry-run") || switch?("n")
   end
 
+  def keep_files?
+    flag? "--keep-files"
+  end
```

Comment:
I think this raises the question what files we're talking about (though that's documented). I think `keep_temp?` and `--keep-temp` would be more self-descriptive.